### PR TITLE
Deprecate `SkyFormat` and `SkyWindowRefService`

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**
Adding deprecation messages in preparation for:
https://github.com/blackbaud/skyux-core/issues/6
https://github.com/blackbaud/skyux-core/issues/5

### CODE REVIEW

File: src/app/public/modules/window/window-ref.ts [hunk 1]
```diff
@@ -2,12 +2,10 @@ import {
   Injectable
 } from '@angular/core';
 
-function getWindow(): any {
+export function getWindow(): any {
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/window/window-ref.ts]: This is needed to satisfy Angular 8 AoT build requirements.

File: src/app/public/modules/format/format.ts [hunk 2]
```diff
@@ -3,6 +3,9 @@ export {
   SkyAppFormat
 } from './app-format';
 
+/**
+ * @deprecated Please use `SkyAppFormat` instead.
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/format/format.ts]: @blackbaud-johnly This message and the one below need your approval, please.

Comment (blackbaud-johnly) [reply src/app/public/modules/format/format.ts]: I'd change this one to just 
```Use `SkyAppFormat` instead.```

File: src/app/public/modules/window/window-ref.service.ts [hunk 3]
```diff
@@ -2,6 +2,9 @@ import {
   Injectable
 } from '@angular/core';
 
+/**
+ * @deprecated Please use `SkyAppWindowRef` instead.
```
Comment (blackbaud-johnly) [inline src/app/public/modules/window/window-ref.service.ts]: I'd change this one to just
```Use `SkyAppWindowRef` instead.```

**Pull request merged**